docs/clients/python-client.md

```diff
@@ -11,13 +11,13 @@ tab_bar: clients
 To use the Samsara SDK, pip install it via:
 
 ```bash
-$ pip install samsara-sdk
+$ pip install samsara_sdk
 ```
 
 To bring it into the repl:
 
 ```python
-> from samsara-sdk import SamsaraClient
+> from samsara_sdk import SamsaraClient
```

Comment:
yea, but that was only becuase I had forgotten to add [this line](https://github.com/samsara/samsara/pull/64/commits/32734b01f392346e31ca77daff9c09850a2fde7d#diff-3adec5d4a0408c92608059ea9f371a31R1)